stories/useFriction.stories.tsx

```diff
@@ -267,3 +263,28 @@ export const FrictionSVG: React.FC = () => {
     </svg>
   );
 };
+
+export const FrictionProgress: React.FC = () => {
+  const progressRef = useRef<HTMLSpanElement>(null);
+
+  useFriction<HTMLDivElement>({
+    from: {
+      opacity: 0,
+    },
+    to: {
+      opacity: 1,
+    },
+    config: {
+      mu: number('mu', 0.1),
+      mass: number('mass', 30),
+      initialVelocity: number('velocity', 10),
+    },
+    onFrame: progress => {
+      if (progressRef.current) {
+        progressRef.current.innerHTML = `${progress.toFixed(2)}`;
+      }
+    },
```

Comment:
@NgoKnows here's how we pull that demo off – essentially, attach a ref to the `span` so we can edit the `innerHTML`. We can't store this in a `state` value, because then we'd be re-rendering this component on every frame.